plugin/src/main/cpp/register_types.cpp

```diff
@@ -441,6 +448,8 @@ void add_plugin_project_settings() {
 	_add_bool_project_setting(project_settings, "xr/openxr/extensions/meta/composition_layer_settings", true);
 	_add_bool_project_setting(project_settings, "xr/openxr/extensions/meta/dynamic_resolution", true);
 	_add_bool_project_setting(project_settings, "xr/openxr/extensions/meta/headset_id", false);
+	_add_bool_project_setting(project_settings, "xr/openxr/extensions/meta/performance_metrics", false);
+	_add_bool_project_setting(project_settings, "xr/openxr/extensions/meta/performance_metrics_enable_on_startup", false);
```

Comment:
Should the "enable on startup" option be `true` by default? If we think this is the most common way that the extension will be used, it seems like it should be? It won't do anything unless the extension itself is enabled, but this would mean one click instead of two

Also, I'm starting to wonder if we shouldn't use a different word than "enable" for this since it is confusing with the idea of enabling the OpenXR module? Maybe the word should be "capture" and so "capture on startup"? And perhaps this should be a suboption, similar to the color space and scene API options?

So, something like:

```suggestion
	_add_bool_project_setting(project_settings, "xr/openxr/extensions/meta/performance_metrics/capture_on_startup", true);
```

Reply:
I agree with these suggestions, and I've applied them in the latest push. 🙂